src/content/docs/apm/agents/manage-apm-agents/agent-data/collect-custom-metrics.mdx

```diff
@@ -107,7 +107,7 @@ Implementing custom metrics requires API calls. The exact details of the API cal
 Collecting too many metric timeslices can impact the performance of both your application and New Relic. For example, if you have thousands of individual users, avoid creating metrics to track the performance of their unique user IDs. This could result in such a large number of metrics that it becomes nearly impossible to navigate or make sense of the data. Instead, use a placeholder, such as an asterisk (\*), instead of individual user IDs.
 
 <Callout variant="important">
-  To avoid potential data problems, try to keep the total number of unique metric timeslices introduced by custom metrics under 2000.
```

Comment:
This recommendation still stands at 2k as its not directly related to the query result limit.